Packs/Synapse/Integrations/Synapse/Synapse.py

```diff
@@ -611,20 +579,15 @@ def file_reputation_command(client, args, good_tag, bad_tag):
             dbot_score=dbot_score
         )
 
-        file_standard_list.append(file_standard_context)
-        file_data_list.append(file_data)
-
-    readable_output = tableToMarkdown('File List', file_data_list)
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('File List', file_data),
```

Comment:
Given a single File, I feel that File List is not a good human-readable title, perhaps File Details?